src/form/Calendar/Calendar.tsx

```diff
@@ -255,19 +273,46 @@ export const Calendar: FC<CalendarProps> = ({
             scale: { type: animated ? 'tween' : false }
           }}
         >
-          {view === 'days' && (
-            <CalendarDays
-              value={viewValue}
-              min={min}
-              max={max}
-              disabled={disabled}
-              isRange={isRange}
-              current={isRange ? [rangeStart, rangeEnd] : date}
-              xAnimation={xAnimation}
-              animated={animated}
-              onChange={dateChangeHandler}
-            />
-          )}
+          {view === 'days' &&
+            (isRange ? (
```

Comment:
I'm not sure we want isRange to be the determining factor on whether to show calendars side by side. we will need to add a prop that users can turn on